woodwork/type_sys/type_system.py

```diff
@@ -145,6 +153,10 @@ def add_type(self, logical_type, inference_function=None, parent=None):
                 Defaults to None. If not specified, this LogicalType will never be inferred.
             parent (LogicalType, optional): The parent LogicalType, if applicable. Defaults to None. If not specified,
                 this type will be considered a root type with no parent.
+            treatment (string, optional): If the specified LogicalType already exists, determine the logic that should
+                be applied. Options are "replace", "ignore", or None (default). "Replace" will unregister the present
```

Comment:
Nit: The options are all lowercase, but the next "Replace", "Ignore" mentioned are capitalized. It'd be cleaner to keep those lower as well.